docs/docsite/rst/reference_appendices/release_and_maintenance.rst

```diff
@@ -1,86 +1,94 @@
-vulnerabilities (*CVE*).
+#. Collection freeze (no new Collections or new versions of existing Collections) on the Ansible community package
+#. Release candidate for Ansible community package, testing, additional release candidates as necessary
+#. Release of a new Ansible community package major version based on the new ansible-core, for example, Ansible 4.0.0 based on ansible-core 2.11
 
-You can refer to the :ref:`porting guides<porting_guides>` for tips on updating your Ansible
-playbooks to run on newer versions. For Ansible 2.10 and later releases, you can install the Ansible package with ``pip``. See :ref:`intro_installation_guide` for details.  For older releases, You can download the Ansible release from `<https://releases.ansible.com/ansible/>`_.
+   * Newest release of the Ansible community package is the only version now maintained
+   * Work on new features continues in Collections
+   * Individual Collections can make multiple minor and/or major releases
+
+#. Minor releases of three maintained ansible-core versions every three weeks (2.11.1)
+#. Minor releases of the single maintained Ansible community package version every three weeks (4.1.0)
+#. Feature freeze on ansible-core
+#. Release candidate for ansible-core, testing, additional release candidates as necessary
+#. Release of the next ansible-core major version, cycle begins again
+
+Release cycle for the Ansible community package
+-----------------------------------------------
+
+The Ansible community team typically releases two major versions of the community package per year, on a flexible release cycle that trails the release of ``ansible-core``. This cycle can be extended to allow for larger changes to be properly implemented and tested before a new release is made available. See :ref:`roadmaps` for upcoming release details. Between major versions we release a new minor version of the Ansible community package every three weeks. Minor releases include new backwards-compatible features, modules and plugins as well as bug fixes.
 
+Starting with version 2.10, the Ansible community team guarantees maintenance for only one major community package release at a time. For example, when Ansible 4.0.0 gets released, the team will stop making new 3.x releases. Community members may maintain older versions if desired.
 
+Older, unmaintained versions of the Ansible community package might contain unfixed security vulnerabilities (*CVEs*). If you are using a release of the Ansible community package that is no longer maintained, we strongly encourage you to upgrade as soon as possible in order to benefit from the latest features and security fixes.
 
+Each major release of the Ansible community package accepts the latest released version of each included Collection and the latest released version of ansible-core. Major releases of the Ansible community package can contain breaking changes in the modules and other plugins within the included Collections and/or in core features.
```

Comment:
The Collection version is fixed at RC time. Only a blocker would allow a collection version update at that point afaik. Ask @abadger for sure.

So maybe the first sentence needs something like "latest released version of each included Collection at the time the release candidate happens". Or something. Dunno - the wording addition helps developers understand when their collection will 'freeze'. Not sure if the added sentence helps users though, so take  this suggestion w/ a grain of salt.

Reply:
Yes, good point. I added a link to the Roadmaps "for specific schedules and deadlines" so this page can stay a little more general.